integrations/make.com/pnpm-lock.yaml

```diff
@@ -0,0 +1,2588 @@
+lockfileVersion: '6.0'
```

Comment:
**chore(blocking):** Remove this file and make sure you are developing your integration in a clone of the `botpress` repository, set up your development environment as described in our [Contributing section](https://github.com/botpress/botpress?tab=readme-ov-file#building-from-sources)! It should look something like this:
1. `git clone https://github.com/botpress/botpress.git`
2. `cd botpress/integrations`
3. `bp init`
4. `cd <my_new_integration_directory>`
5. `pnpm install`

When you do it this way, the pnpm install command should modify the root `pnpm-lock.yaml` instead of creating a new one!